akka-remote/src/main/scala/akka/remote/artery/Codecs.scala

```diff
@@ -303,6 +331,8 @@ private[remote] class Decoder(
             envelope,
             association)
 
+          instrument.remoteMessageReceived(recipient, deserializedMessage, sender, headerBuilder.metadata)
```

Comment:
how can this compile? `deserializedMessage` is in `Deserializer` stage.
Would be good to move to the `Deserializer` stage anyway, because that is done in the parallel inbound lanes